onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java

```diff
@@ -1545,13 +1545,14 @@ private void customizeRequests(AbstractTableInfo ti)
 //        updateCol.setHidden(true);
 //        updateCol.setDisplayWidth("150");
     }
-
+//Update by jonesga 2021-10-01
+    //update to item below to change the setScale to 200 for protocol tritle
     private void customizeProtocol(AbstractTableInfo ti)
     {
         ti.getMutableColumn("inves").setHidden(true);
         ti.getMutableColumn("maxAnimals").setHidden(true);
-        ti.getMutableColumn("title").setScale(80);
-        ti.getMutableColumn("investigatorId").setHidden(false);
+        ti.getMutableColumn("title").setScale(200);
+        Update of ONPRC_EHRCustomizer to allow 200 characters for Title of Protocol        ti.getMutableColumn("investigatorId").setHidden(false);
```

Comment:
I'm sure this is intended to be a comment but that's now how it's formatted and it's breaking the build. Please correct.

Reply:
CHanges made and submitted to Lakshmi for review - Thanks